rancid-tomatillos/src/containers/Login/Login.js

```diff
@@ -48,23 +48,23 @@ class Login extends Component {
 
 
 
-  render() { 
+  render() {
     {
       if (this.props.loggedInStatus) {
         return (
       <Redirect to='/' />
         )
       } else {
 
-        return ( 
+        return (
           <form>
             <label htmlFor='email' className="form-label">Email</label>
-            <input id='email' className="form-input" type="text" name="email" 
-            value={this.state.email} onChange={(e) => this.handleChange(e)} placeholder="BigTimeTimmyJim@yahoo.com" autocomplete='off' />
+            <input id='email' className="form-input" type="text" name="email"
+            value={this.state.email} onChange={(e) => this.handleChange(e)} placeholder="BigTimeTimmyJim@yahoo.com" />
             <label htmlFor='password' className="form-label">Password</label>
```

Comment:
Alright, I will concede on this, the repetitive logging in is a pain.

Reply:
Also think we should change the placeholder name to something that is more helpful, @ckoga thoughts?

Reply:
When you say more helpful do you mean an example that is closer to what the user will actually put in?  We could always put in ```email``` or ```example@something.com``` but that is similar to what we have.  I'm open to suggestions but also don't think what we have is bad, it could be shorter I guess.

Reply:
Yea, I think that something like you just listed would be great!